src/GraphQL/Types/Collections/ManualGraphTypeMappingResolver.cs

```diff
@@ -0,0 +1,20 @@
+namespace GraphQL.Types.Collections;
+
+internal class ManualGraphTypeMappingResolver : IGraphTypeMappingResolver
```

Comment:
internal or public?

Reply:
Schema still has methods to add CLR to graph type mappings without making this public

Reply:
internal